packages/@ourworldindata/grapher/src/tooltip/Tooltip.tsx

```diff
@@ -1,8 +1,56 @@
 import React from "react"
+import classnames from "classnames"
 import { observable, computed, action } from "mobx"
 import { observer } from "mobx-react"
-import { Bounds } from "@ourworldindata/utils"
-import { TooltipProps, TooltipManager } from "./TooltipProps"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome/index.js"
+import { faInfoCircle } from "@fortawesome/free-solid-svg-icons"
+import { Bounds, PointVector } from "@ourworldindata/utils"
+import { TooltipProps, TooltipManager, TooltipFadeMode } from "./TooltipProps"
+export * from "./TooltipContents.js"
+
+export const TOOLTIP_FADE_DURATION = 400 // $fade-time + $fade-delay in scss
+
+export class TooltipState<T> {
```

Comment:
The name `TooltipState` might be a bit too broad for what this class does. Can we find a more descriptive name? It also confused me for a minute that `position` is a property of this class but isn't managed here. Perhaps removing `position` from this class would also make it easier to find a more descriptive name for it?